ui/src/app/settings/selectors/config.js

```diff
@@ -0,0 +1,119 @@
+import { createSelector } from "reselect";
+
+const config = {};
+
+/**
+ * Returns MAAS config
+ * @param {Object} state - The redux state.
+ * @returns {Array} A list of all state.config.items.
+ */
+config.get = state => state.config.items;
```

Comment:
@Caleb-Ellis I think we've decided to call this `all` to distinguish from a `get` selector which takes a prop as a param.